prometheus/generic/setup-haproxy-exporter/README.md

```diff
@@ -0,0 +1,57 @@
+setup-node-exporter
```

Comment:
`s/node/haproxy/`

Reply:
Oops, I fixed this to setup-haproxy-exporter